### PR TITLE
use default values of cluster-cloud-director chart:v0.50.0

### PR DESCRIPTION
**Pull request by vxav**
### What this PR does

In `cluster-cloud-director` `v0.50.0`, we move to a release model where K8s version and VM template are set in default values to be controlled by Giant Swarm through cluster app versions.

Towards https://github.com/giantswarm/roadmap/issues/3392

### Checklist

- [x] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Review by erkanerol — COMMENTED** [review]
LGTM

File: providers/capvcd/standard/test_data/cluster_values.yaml [hunk 1]
```diff
@@ -1,11 +1,8 @@
 baseDomain: test.gigantic.io
 controlPlane:
-  catalog: giantswarm
   replicas: 3
+  diskSizeGB: 30
```
Comment (AverageMarcus) [inline providers/capvcd/standard/test_data/cluster_values.yaml:4]: How come this is needed for standard but not upgrade suite?

Comment (vxav) [reply providers/capvcd/standard/test_data/cluster_values.yaml:4]: It was an oversight on my part, thanks.
Fixed

Comment (AverageMarcus) [reply providers/capvcd/standard/test_data/cluster_values.yaml:4]: 😕  I still see it on the PR. Is it just me?

Comment (AverageMarcus) [reply providers/capvcd/standard/test_data/cluster_values.yaml:4]: Oh, you need it on both! Got it!

**Pull request merged**